src/utils/openwith.js

```diff
@@ -102,8 +100,7 @@ export default class OpenWith {
     static getOpenWithLinkParams(image_id, image_name, iviewer_url) {
         let ret = [];
```

Comment:
The ret variable can go now since you are returning the result of map and filter instead of the iteration I had